frontend/src/scenes/funnels/FunnelHistogram.tsx

```diff
@@ -5,58 +5,52 @@ import { humanFriendlyDuration, humanizeNumber } from 'lib/utils'
-    const { stepsWithCount, stepReference, histogramStep, histogramStepsDropdown } = useValues(funnelLogic)
+export function FunnelHistogramHeader(): JSX.Element | null {
+    const { stepsWithCount, stepReference, histogramStepsDropdown } = useValues(funnelLogic)
     const { changeHistogramStep } = useActions(funnelLogic)
+    const { allFilters } = useValues(insightLogic)
+
+    if (allFilters.display !== FUNNELS_TIME_TO_CONVERT) {
+        return null
+    }
 
     return (
-        <>
-            <div className="funnel__header__info">
-                Mean time:{' '}
-                <span className="l4" style={{ color: 'var(--primary)' }}>
-                    {humanFriendlyDuration(stepsWithCount[histogramStep]?.average_conversion_time)}
-                </span>
-            </div>
-            <div className="funnel__header__steps">
-                <span className="funnel__header__steps__label">Steps</span>
-                {histogramStepsDropdown.length > 0 && (
-                    <Select
-                        defaultValue={histogramStepsDropdown[0]?.value}
-                        onChange={changeHistogramStep}
-                        dropdownMatchSelectWidth={false}
-                        data-attr="funnel-bar-layout-selector"
-                        optionLabelProp="label"
-                    >
-                        {histogramStepsDropdown.map((option, i) => {
-                            const basisStep = getReferenceStep(stepsWithCount, stepReference, i)
-                            return (
-                                <Select.Option
-                                    key={option?.value}
-                                    value={option?.value || 1}
-                                    label={<>{option?.label}</>}
-                                >
-                                    <Col style={{ minWidth: 300 }}>
-                                        <Row style={{ justifyContent: 'space-between', padding: '8px 0px' }}>
-                                            <span className="l4">{option?.label}</span>
-                                            <span className="text-muted-alt">
-                                                Mean time: {humanFriendlyDuration(option?.average_conversion_time)}
-                                            </span>
-                                        </Row>
-                                        <Row className="text-muted-alt">
-                                            Total conversion rate:{' '}
-                                            {humanizeNumber(calcPercentage(option.count, basisStep.count))}%
-                                        </Row>
-                                    </Col>
-                                </Select.Option>
-                            )
-                        })}
-                    </Select>
-                )}
-            </div>
-        </>
+        <div className="funnel__header__steps">
```

Comment:
Because the "Mean time" info was moved to the `FunnelCanvasLabel`, the change here is just with the indentation of each row.